index.js

```diff
@@ -3,6 +3,7 @@ const url = require('url')
 const querystring = require('querystring')
 const axios = require('axios')
 const { send } = require('micro')
+const ghURL = process.env.GH_HOST || 'github.com'
```

Comment:
Will do. My logic in calling it `githubURL` was to make it consistent in naming style used elsewhere, such as `createRedirectHTML`. `githubUrl` works for me.